### PR TITLE
Use pj for X509 and PKCS

### PR DESCRIPTION
**Pull request by Maijin**
Use pj for X509 and PKCS #12870

### CODE REVIEW

File: libr/bin/p/bin_pe.c [hunk 1]
```diff
@@ -108,9 +108,8 @@ static char *signature (RBinFile *bf, bool json) {
 	}
 	struct PE_ (r_bin_pe_obj_t) * bin = bf->o->bin_obj;
 	if (json) {
-		RJSVar *json = r_pkcs7_cms_json (bin->cms);
-		char *c = r_json_stringify (json, false);
-		r_json_var_free (json);
+		PJ *pj = r_pkcs7_cms_json (bin->cms);
+		char *c = pj_string (pj);
```
Comment (wargio) [inline libr/bin/p/bin_pe.c]: does PJ require to be freed before returning?

File: libr/util/pkcs7.c [hunk 2]
```diff
@@ -5,12 +5,12 @@
 #include <r_util.h>
 #include "./x509.h"
 
-extern RJSVar *r_x509_name_json (RX509Name* name);
+extern RJSVar *r_x509_name_json (PJ *pj, RX509Name *name);
```
Comment (wargio) [inline libr/util/pkcs7.c]: you are still returning `RJSVar`

File: libr/bin/p/bin_pe.c (hunk 1 above)
Comment (wargio) [reply libr/bin/p/bin_pe.c]: i've checked the usage of it.
you forgot to free it.

Comment (Maijin) [reply libr/bin/p/bin_pe.c]: If I free it there I got:

```
r2(69885,0x1001295c0) malloc: *** error for object 0x103024000: pointer being freed was not allocated
r2(69885,0x1001295c0) malloc: *** set a breakpoint in malloc_error_break to debug
```

Comment (Maijin) [reply libr/bin/p/bin_pe.c]: Where should I free it?

Comment (wargio) [reply libr/bin/p/bin_pe.c]:
```c
PJ *pj = r_pkcs7_cms_json (bin->cms);
char *c = pj_string (pj);
pj_free (pj);
return c;
```

File: libr/bin/p/bin_pe.c [hunk 3]
```diff
@@ -108,10 +108,13 @@ static char *signature (RBinFile *bf, bool json) {
 	}
 	struct PE_ (r_bin_pe_obj_t) * bin = bf->o->bin_obj;
 	if (json) {
-		RJSVar *json = r_pkcs7_cms_json (bin->cms);
-		char *c = r_json_stringify (json, false);
-		r_json_var_free (json);
-		return c;
+		PJ *pj = r_pkcs7_cms_json (bin->cms);
+		char *json_str = strdup((char *)pj_string(pj));
```
Comment (wargio) [inline libr/bin/p/bin_pe.c]: move `char *json_str = strdup((char *)pj_string(pj));` inside the `if`.
after that, i'll merge

Comment (Maijin) [reply libr/bin/p/bin_pe.c]: yep done

**Comment by wargio** [comment]
ready to merge when travis & app veyor will be green :)

**Pull request merged**